src/datacatalog_tag_manager/tag_datasource_processor.py

```diff
@@ -27,18 +27,43 @@ def create_tags_from_csv(self, file_path):
 
         logging.info('')
         logging.info(f'Creating the Tags...')
-        created_tags = self.__create_tags_from_dataframe(dataframe)
+        created_tags = self.__execute_datacatalog_tags_method_on_dataframe(
+            dataframe, datacatalog_tags_method=self.__datacatalog_facade.create_or_update_tag)
 
         logging.info('')
         logging.info('==== Create Tags from CSV [FINISHED] =============')
 
         return created_tags
 
-    def __create_tags_from_dataframe(self, dataframe):
+    def delete_tags_from_csv(self, file_path):
+        """
+        Delete Tags by reading information from a CSV file.
+
+        :param file_path: The CSV file path.
+        :return: A list with all Tags created.
+        """
+        logging.info('')
+        logging.info('===> Delete Tags from CSV [STARTED]')
+
+        logging.info('')
+        logging.info('Reading CSV file: %s...', file_path)
+        dataframe = pd.read_csv(file_path)
+
+        logging.info('')
+        logging.info(f'Deleting the Tags...')
+        deleted_tag_names = self.__execute_datacatalog_tags_method_on_dataframe(
+            dataframe, datacatalog_tags_method=self.__datacatalog_facade.delete_tag)
```

Comment:
Do we need `datacatalog_tags_method=`? Since it's mandatory, I believe suppressing it makes the code clearer.